packages/models-library/src/models_library/function_services_catalog/services/probes.py

```diff
@@ -66,7 +66,30 @@ def create_metadata(type_name: str, prefix: str | None = None) -> ServiceDockerD
     }
 )
 
+META_FILE = ServiceDockerData.parse_obj(
+    {
+        "integration-version": LATEST_INTEGRATION_VERSION,
+        "key": f"{FUNCTION_SERVICE_KEY_PREFIX}/iterator-consumer/probe/file",
+        "version": "1.0.0",
+        "type": ServiceType.FRONTEND,
+        "name": "File probe",
+        "description": "Probes its input for files",
+        "thumbnail": create_fake_thumbnail_url("file"),
+        "authors": [
+            OM,
```

Comment:
Maybe WVG?

Reply:
Ah ok, was not sure if it was referring to a real author, or that it meant 'osparc maintenaner'. Do I need to add a key with WVG somewhere in another file in the repo?

Reply:
It seems to be pointing some environment variables after a couple indirections, but then i don't see anymore how these are set. @odeimaiz , you know by chance?